__tests__/application/respond.test.js

```diff
@@ -577,13 +592,14 @@ describe('app.respond', () => {
 
       app.use(ctx => {
         ctx.body = new Response(null, { status: 200, statusText: 'OK' })
+        console.log(ctx)
       })
 
       return request(app.callback())
-        .head('/')
+        .get('/')
         .expect(200)
         .expect('content-type', 'application/octet-stream')
-        .expect('content-length', '2')
+        .expect(Buffer.from([]))
```

Comment:
Why has the expectation changed?

Reply:
method changed from `head` to `get` to increase test coverage

it also changed the result